### ISSUE TITLE
What kind of packages are accepted?

### ISSUE THREAD

**Issue by egoist** [vuejs/vue-curated#27]
I mean some requirements like minimal stars, project activity, code quality, etc...

**Comment by nickmessing** [comment]
I can suggest something like at least 2 approves after some basic tests by someone from the team. I have a lot of free time for that.

**Comment by posva** [comment]
I think we should close prs until we find out some reasoning when accepting packages or we'll end up doing the same mistakes we did with vue-awesome

**Comment by yyx990803** [comment]
Yes, we should stop merging PRs for now - and I may need to revert a few ones that have already been merged, I've just been to busy to properly review them. The whole purpose of this project is curation so a formal requirement / evaluation process is needed.

**Comment by Justineo** [comment]
Hi guys, any progress on this?

**Comment by Akryum** [comment]
My thoughts so far:

## Minimal Requirements

- Find and explain a common use case (shouldn't be too 'niche')
- Have a demo repository to showcase it using this package only (focused demo)
- Have a good enough documentation
  - We should discuss what's a 'good enough' doc
  - At least in English
- If an npm package, it should be correctly compiled for an easy usage with at least webpack (and optionally the browser)
- Have active maintainers on the repository
- Get a core Vue member/contributor to test the demo project and verify it's a good solution for the use case

## Other Potential Requirements

- Get a core Vue member/contributor to review the code and suggest improvements

**Comment by yyx990803** [comment]
@Akryum looks good to me! I'd say we should run the current list through this and see if there are any that needs to be improved/removed.

**Comment by Akryum** [comment]
Maybe we should create issues in the corresponding repositories?

**Comment by Akryum** [comment]
https://github.com/vuejs/vue-curated/blob/master/CONTRIBUTING.md (PR Welcome!)

**Comment by guyulmaz** [comment]
Maybe vuetify can be added to this list. quasar and vue-metarial is existing, but vuetify is not?

**Comment by tmorehouse** [comment]
Maybe since curation is not happening, and the last merge was some timeago, maybe the Vue team should just close this Repo and remove the Vue Curated link fro the Vue docs and un-publish the site?
